### PR TITLE
--help: Add a note on using IPFS_PATH to the footer of the helptext.

### PR DESCRIPTION
**Pull request by sahib**
I couldn't find this piece of information in the official docs, just scattered over a few issues on GitHub.

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 86b2081ffcc19e1733b7901fb9356eb514337440
  * Invalid signoff. Commit message must end with  
    **License: MIT  
Signed-off-by: .* <.*>**

[Guidelines](https://github.com/ipfs/community/blob/master/contribution-guidelines.md#commit-messages) and [a script](https://github.com/ipfs/community/blob/master/dev/tools/hooks/setup_commit_msg_hook.sh) are available to help. Your feedback on GitCop is welcome on [this issue](https://github.com/ipfs/infrastructure/issues/23).

---
This message was auto-generated by https://gitcop.com

File: core/commands/root.go [hunk 1]
```diff
@@ -69,6 +69,9 @@ TOOL COMMANDS
     commands      List all available commands
 
 Use 'ipfs <command> --help' to learn more about each command.
+
+If you want to operate on a different repository than the default at ~/.ipfs,
+then prefix any ipfs command with IPFS_PATH=/path.
```
Comment (whyrusleeping) [inline core/commands/root.go]: I would phrase this a little different, to give the reader a more clear idea of whats going on:

"You may set the `IPFS_PATH` environment variable to make ipfs use a different location for its on disk repo."

Or something, cc @jbenet @RichardLitt

Comment (RichardLitt) [inline core/commands/root.go]: @sahib This is a great add! But I agree, it is worded a bit strange. How about:

> If you want IPFS to use a different directory locally than the default  `~/.ipfs`, set the `IPFS_PATH` environment variable to point to your preferred repository.

Sahib, do you know how to amend commits?

Comment (sahib) [inline core/commands/root.go]: I used your suggestion and updated the pull request.

> Sahib, do you know how to amend commits?

I just squashed my commits to one.
(Had to use ``git push --force`` but I guess nobody used that commit yet...)

**Comment by RichardLitt** [comment]
LGTM!

File: core/commands/root.go (hunk 1 above)
Comment (RichardLitt) [inline core/commands/root.go]: :+1:

File: core/commands/root.go [hunk 2]
```diff
@@ -69,6 +69,9 @@ TOOL COMMANDS
     commands      List all available commands
 
 Use 'ipfs <command> --help' to learn more about each command.
+
+If you want IPFS to use a different directory locally than the default ~/.ipfs,
+set the IPFS_PATH environment variable to point to your preferred directory.
```
Comment (jbenet) [inline core/commands/root.go]: How about this (+ example):

```

ipfs uses a repo (repository) in the local file system. By default, the repo is located
at ~/.ipfs. To change the repo location, set the $IPFS_PATH environment variable:

    export IPFS_PATH=/path/to/ipfsrepo
```

**Comment by jbenet** [comment]
LGTM thanks-- one minor edit above. sorry for the delay @sahib

**Comment by sahib** [comment]
@jbenet: Changed. Only change from your suggestion was spelling out *repository* long the first time and abbreviating afterwards.

**Comment by RichardLitt** [comment]
lgtm. Thanks.

**Comment by jbenet** [comment]
thanks @sahib !

**Pull request merged**